tests/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Set Up Google Tests
+
+# build testing infrastructure
+enable_testing()
+
+# Download and unpack googletest at configure time
+configure_file(CMakeLists.txt.in googletest-download/CMakeLists.txt)
+execute_process(COMMAND ${CMAKE_COMMAND} -G "${CMAKE_GENERATOR}" .
+  RESULT_VARIABLE result
+  WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/googletest-download )
+if(result)
+  message(FATAL_ERROR "CMake step for googletest failed: ${result}")
+endif()
+execute_process(COMMAND ${CMAKE_COMMAND} --build .
+  RESULT_VARIABLE result
+  WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/googletest-download )
+if(result)
+  message(FATAL_ERROR "Build step for googletest failed: ${result}")
+endif()
+
+# Add googletest directly to our build. This defines
+# the gtest and gtest_main targets.
+add_subdirectory(${CMAKE_CURRENT_BINARY_DIR}/googletest-src
+                 ${CMAKE_CURRENT_BINARY_DIR}/googletest-build
+                 EXCLUDE_FROM_ALL)
+
+
+# Define C macros for available_solvers
+# used in the parametrized tests so that it knows which
+# solvers to run with
+if (BUILD_BTOR)
+  add_definitions(-DBUILD_BTOR)
+endif(BUILD_BTOR)
+
+if (BUILD_CVC4)
+  add_definitions(-DBUILD_CVC4)
+endif(BUILD_CVC4)
+
+if (BUILD_MSAT)
+  add_definitions(-DBUILD_MSAT)
+endif(BUILD_MSAT)
+
+add_library(available-solvers "${SMT_SWITCH_LIB_TYPE}"
+  "${PROJECT_SOURCE_DIR}/tests/available_solvers.cpp")
+
+target_include_directories(available-solvers PUBLIC "${PROJECT_SOURCE_DIR}/tests/")
+# TODO: Fix this
+target_link_libraries(available-solvers smt-switch ${EXTRA_LIBS})
```

Comment:
What is this TODO and what is EXTRA_LIBS?

Reply:
Oops, good catch! I addressed it with 5369eff91efb3bd01fa72f5d06c2f364591c0b34. The todo was just left over, and I gave EXTRA_LIBS a better name. It's the list of solver backend libraries that are needed.